frontend/src/components/PageHeader.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components/macro'
+
+export default function PageHeader({headerText}) {
+    return (
+        <HeaderWrapper>
+            {headerText}
+        </HeaderWrapper>
+    )
+}
+
+const HeaderWrapper = styled.section`
+  background: var(--color-turquoise-green);
+  font-family: Halant;
+  font-size: 30px;
+  padding: 20px 20px 20px 45px;
+  color: var(--color-cultured)
+`
```

Comment:
I would make the section to a header element and simply return the styled header by default
```suggestion
import styled from 'styled-components/macro'

export default styled.header`
  background: var(--color-turquoise-green);
  font-family: Halant;
  font-size: 30px;
  padding: 20px 20px 20px 45px;
  color: var(--color-cultured)
`
```

Reply:
but how would I pass the header text if the component is only a styled component? I intend on using it again later on with different header texts

Reply:
got it - no self-closing tag 🤣  that's what you get for working late